src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -172,6 +181,91 @@ public static void createTempPartitionedSourceBQTable() throws IOException, Inte
     BeforeActions.scenario.write("BQ Source Table " + bqSourceTable + " created successfully");
   }
 
+  /**
+   * Create BigQuery table with different list of dataypes (transaction_info BOOL, transaction_num BYTES,
+   * transaction_uid STRING, transaction_date DATE, transaction_dt DATETIME, transaction_time TIME, latitude FLOAT64,
+   * unique_key INT64, business_ratio NUMERIC, updated_on TIMESTAMP, parent ARRAY<STRING>,
```

Comment:
Looks like bignumeric is missing. Please make sure we have coverage for all supported data types in source and sink.
https://github.com/data-integrations/google-cloud/blob/develop/docs/BigQueryTable-batchsource.md#data-type-mappings-from-bigquery-to-cdap
https://github.com/data-integrations/google-cloud/blob/develop/docs/BigQueryTable-batchsink.md#data-type-mappings-from-cdap-to-bigquery

Reply:
Changes done.